data/countries2.csv

```diff
@@ -123,7 +123,7 @@ MWI,,,0,sub-saharan-africa,,not-oecd,"icescr,iccpr",
 MYS,,,0,east-asia-pacific,east-asia,"asean,oic,not-oecd",,
 MDV,,,0,south-asia,,"oic,not-oecd","icescr,iccpr",
 MLI,,,0,sub-saharan-africa,,"oic,not-oecd","icescr,iccpr",
-MLT,,,1,middle-east-north-africa,europe,not-oecd,"icescr,iccpr",
+MLT,,,1,middle-east-north-africa,north-africa,not-oecd,"icescr,iccpr",
```

Comment:
@annemariebrook when implementing the subregion filtering, I noticed that Malta was classified to be in the `middle-east-north-africa` region but in the `europe` subregion. To be consistent, also with the [live version](https://github.com/HumanRightsMeasurementInitiative/hrmi-dataportal-data/blob/master/data/countries.csv#L156), I have changed its subregion to `north-africa` (in the development version)

Reply:
@tmfrnz thanks very much for spotting this inconsistency. I just read up a bit more on Malta and learned that they are a member of the EU, and they also have the Euro as their currency, so I think we should classify them as being in the Europe region. Sound OK? Can you please make this change?

Reply:
Agreed and done both in future (https://github.com/HumanRightsMeasurementInitiative/hrmi-dataportal-data/pull/9/commits/0bb71f69d9937044a0777c0d3cc4f2eb76057ce7) and current 
(https://github.com/HumanRightsMeasurementInitiative/hrmi-dataportal-data/commit/ee1f6bdf8fd93e84be56787b6e43dee494b9fc58) versions